test/acceptance/features/rabbit.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Rabbit client
+
+  @rabbit
+  Scenario: Publish and subscribe a text message
+    Given I generate a UUID and store it in context "CorrelationId"
+    Given the rabbit endpoint "amqp://guest:guest@localhost:5672/"
+      And I subscribe to the rabbit topic "test-rabbit-text-properties-topic"
+      And I set standard rabbit properties
+          | ContentType   | text/plain           |
+          | CorrelationId | [CTXT:CorrelationId] |
+     When I publish a message to the rabbit topic "test-rabbit-text-properties-topic" with the text
+          """
+          This is a test message
+          """
+     Then I wait up to "3" seconds for a rabbit message with the standard properties
+          | ContentType   | text/plain           |
+          | CorrelationId | [CTXT:CorrelationId] |
+      And the rabbit message body has the text
+          """
+          This is a test message
+          """
+      And I unsubscribe from the rabbit topic "test-rabbit-text-properties-topic"
+
+  @rabbit
+  Scenario: Publish and subscribe a JSON message
+    Given I generate a UUID and store it in context "CorrelationId"
+    Given the rabbit endpoint "amqp://guest:guest@localhost:5672/"
+      And I subscribe to the rabbit topic "test-rabbit-json-properties-topic"
+      And I set standard rabbit properties
+          | ContentType   | application/json     |
+          | CorrelationId | [CTXT:CorrelationId] |
```

Comment:
IMHO, I do not think it is needed to establish again the rabbit standard properties along as this featured may be tested in other scenario. Also, the property `ContentType` in this case will be overwritten in the step to send a JSON message:
https://github.com/Telefonica/golium/blob/36764ff47170fc520c74fa41126b1fefcb00f562/steps/rabbit/session.go#L209

In addition, if no ContentType is established by the standard properties step, its value is set to `text/plain` when the publishing struct is builded:
https://github.com/Telefonica/golium/blob/36764ff47170fc520c74fa41126b1fefcb00f562/steps/rabbit/session.go#L192-L194